src/Plate/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { range, uniq } from 'lodash';
+import React, { ReactNode } from 'react';
+
+import { MLL_THEME } from '../theme';
+
+export type PlateProps = {
+  data?: Array<{
+    position: {
+      row: 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H';
+      column: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+    };
+    content?: string | ReactNode;
+    selected?: boolean;
+  }>;
+};
+
+const TUBE_COUNT = 96;
+const CELLS = range(1, TUBE_COUNT + 1);
+const COLUMNS = range(1, 12 + 1);
+const ROWS = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'];
+
+const generalLineStyle = {
+  padding: '8px 4px',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+};
+
+export function Plate(props: PlateProps) {
+  if (
+    props.data &&
+    uniq(
+      props.data.map(
+        (wellData) => wellData.position.row + wellData.position.column,
+      ),
+    ).length !== props.data.length
+  ) {
+    throw new Error(
+      'Property "data" contains a records with non-unique position',
+    );
+  }
+
+  return (
+    <>
+      <div
+        style={{
+          display: 'grid',
+          gridTemplateColumns:
+            '1fr 4fr 4fr 4fr 4fr 4fr 4fr 4fr 4fr 4fr 4fr 4fr 4fr',
+          gridGap: '3px',
+        }}
+      >
+        <span style={generalLineStyle} />
+        {COLUMNS.map((position) => (
+          <span style={generalLineStyle} key={`columnLabel${position}`}>
+            <strong>{position}</strong>
+          </span>
+        ))}
+        {CELLS.map((position) => (
+          <Well key={position} position={position} data={props.data} />
+        ))}
+      </div>
+    </>
+  );
+}
+
+function Well(props: { position: number; data: PlateProps['data'] }) {
+  const getRowByPosition = (position: number) =>
```

Comment:
Should be a free standing function, since it does not depend on `props`, and not have the useless prefix `get`.

The same goes for the other closures defined in this component.